src/sentry/api/endpoints/event_apple_crash_report.py

```diff
@@ -36,7 +36,7 @@ def get(self, request, event_id):
 
         self.check_object_permissions(request, event.group)
 
-        Event.objects.bind_nodes([event], 'data')
+        Event.objects.bind_nodes([event], 'node_data')
```

Comment:
This is going to be an issue. I much rather change `bind_nodes` so that it accepts the db column in addition to the descriptor and not change all calls to bind_nodes. Otherwise we need to touch some other stuff as well (hipchat-ac plugin though we can delete that, getsentry commands).

Reply:
Can you clarify what you mean with db column vs descriptor please?

Reply:
Reading the other comment, I think I get it now. That would mean that everyone accessing data would have to change to a new field then, instead of continuing to use `event.data`. I'm afraid that'd be a higher risk of breaking something. 

We could probably also patch this in `Event.objects.bind_nodes` and add a special case.

Reply:
@jan-auer i was thinking of letting `bind_nodes` also take the db_column into account as a fallback.